src/ExpressionEvaluator/Core/Test/ResultProvider/Debugger/MemberInfo/TypeImpl.cs

```diff
@@ -439,6 +439,11 @@ protected override bool IsPrimitiveImpl()
             throw new NotImplementedException();
         }
 
+        public override bool IsAssignableTo(Type c)
```

Comment:
@tmat is this the right fix?

Reply:
IDK :(

Reply:
@plnelson

Reply:
Looks like this is used only by tests? If so, I assume it's fine if all tests passed.

Reply:
The debugger implementation is: `return c?.IsAssignableFrom(this) ?? false;`

Reply:
OK, I can replace the throw with this in a follow up PR

Reply:
@plnelson Any reason why the method on the base type doesn't do that?

Reply:
@tmat No.  The fact Roslyn has to add a duplicate implementation shows it would have been better to put the implementation in the base class.

Reply:
Well, our implementation of `IsAssignableFrom` also throws, not sure adding an implementation for `IsAssignableTo` would make much of a difference.